src/routes/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Film'
+export * from './PageNotFound'
```

Comment:
Hmmmm, is it standard practice to have an index file for each folder of components?

Reply:
No clue. But I like it, as it can expose other components so you do not have to specify exactly where they are from.